### PR TITLE
Change VSP to Platform in Collaboration Cycle documentation 

### PR DESCRIPTION
**Pull request by peggygannon**
This PR is for replacing "VSP"  with "Platform" in the Collaboration Cycle github pages:

### CODE REVIEW

**Comment by peggygannon** [comment]
Hi, @shiragoodman, can you review this PR to check all references to VSP have been changed to Platform in our Collaboration Cycle documents? 

The privacy and security template has moved to va.gov-team-sensitive repo, so I'll open another PR for those updates. Thank you!

File: .github/ISSUE_TEMPLATE/analytics-implementation-and-qa-request-template.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ An example of a completed request template can be found [here](https://github.co
 - [ ] [All items of checklist have been executed](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsp/teams/insights-analytics/customer-support-guide-v2.md#%EF%B8%8F-checklist-before-closing-any-ticket)
 - [ ] QA has been completed
 - [ ] Events have been added to `Product` and `Benefit Hub` Content Groups
-- [ ] VFS team completes brief [VSP Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey
+- [ ] VFS team completes brief [Platform Collaboration Cycle Feedback](https://adhoc.optimalworkshop.com/questions/20260uu8-0-0/questions/before) survey
```
Comment (shiragoodman) [inline .github/ISSUE_TEMPLATE/analytics-implementation-and-qa-request-template.md:96]: we can probably delete this line completely. I don't think this survey is still active.

File: platform/working-with-vsp/vsp-collaboration-cycle/README.md [hunk 2]
```diff
@@ -24,10 +24,10 @@ When you’re ready to start the Collaboration Cycle process, submit our kickoff
...
 
-Once submitted, VSP will comment on the ticket with the recommended Collaboration Cycle touchpoints, as well as what practice areas will be involved in your reviews. Those recommended touchpoints will also be added to our tracking spreadsheet (linked below) for reference. 
+Once submitted, we'll comment on the ticket with the recommended Collaboration Cycle touchpoints, as well as what practice areas will be involved in your reviews. Those recommended touchpoints will also be added to our tracking spreadsheet (linked below) for reference. 
 
 ### 3. Prepare for and schedule your touchpoints
-When you’re ready to schedule your recommended touchpoints, review the guidelines outlined in each of the touchpoints below to prepare. Then use our VSP Collaboration Meeting Calendar (linked below) to find a time.  
+When you’re ready to schedule your recommended touchpoints, review the guidelines outlined in each of the touchpoints below to prepare. Then use our Platform Collaboration Meeting Calendar (linked below) to find a time.
```
Comment (shiragoodman) [inline platform/working-with-vsp/vsp-collaboration-cycle/README.md:30]: Meetings*

**Review by shiragoodman — APPROVED** [review]

File: platform/working-with-vsp/vsp-collaboration-cycle/README.md (hunk 2 above)
Comment (peggygannon) [reply platform/working-with-vsp/vsp-collaboration-cycle/README.md]: ok, thanks!

**Pull request merged**